src/components/CallToAction/CallToAction.tsx

```diff
@@ -1,43 +1,41 @@
 import "./CallToAction.scss";
 
 import { t, Trans } from "@lingui/macro";
-import { Box, Button, SvgIcon, Typography } from "@material-ui/core";
+import { Box, SvgIcon, Typography } from "@material-ui/core";
+import { PrimaryButton, SecondaryButton } from "@olympusdao/component-library";
 
 import { ReactComponent as ArrowUp } from "../../assets/icons/arrow-up.svg";
 
-export const LearnMoreButton = () => (
-  <Button
-    variant="outlined"
-    color="secondary"
-    href="https://docs.olympusdao.finance/main/basics/migration"
-    target="_blank"
-    className="learn-more-button"
-  >
-    <Typography variant="body1">
-      <Trans>Learn More</Trans>
-    </Typography>
-
-    <SvgIcon component={ArrowUp} color="primary" />
-  </Button>
-);
```

Comment:
ser, have you visually verified your changes? This branch has compiling errors, and once resolved this button looks like this. Can you please provide sceenshots with your changes to visually verify your changes? 
![image](https://user-images.githubusercontent.com/95196612/152709967-bef3cb0b-a18d-487f-a5b1-c29a6fda5fed.png)

Reply:
Hi @brightiron, This button wasn't visible to me because I didn't have any assets. I have changed this according to the material UI properties.